src/main/java/org/skife/jdbi/v2/unstable/BindIn.java

```diff
@@ -90,21 +100,98 @@ public void apply(SQLStatement q) throws SQLException
     class BindingFactory implements BinderFactory<BindIn>
     {
         @Override
-        public Binder build(BindIn in)
+        public Binder build(final BindIn in)
         {
             final String key = in.value();
 
-            return new Binder<Annotation, Iterable>()
+            return new Binder<Annotation, Object>()
             {
                 @Override
-                public void bind(SQLStatement q, Annotation bind, Iterable coll)
+                public void bind(final SQLStatement q, final Annotation bind, final Object arg)
                 {
+                    final Object[] arr = Util.toArray(arg);
+
                     int idx = 0;
-                    for (Object s : coll) {
+                    for (final Object s : arr)
+                    {
                         q.bind("__" + key + "_" + idx++, s);
                     }
                 }
             };
         }
     }
+
+    final class Util
```

Comment:
isn't an inner class on an interface public static by default? Intellij points out that static is redundant when I try it, and classes outside the package can see BindIn.Util (suggesting the implicit public)